contents/docs/surveys/troubleshooting.mdx

```diff
@@ -53,3 +53,14 @@ You can call `getActiveMatchingSurveys` in your browser console to see which sur
 ```js
 posthog.getActiveMatchingSurveys(surveys => console.log(surveys))
 ```
+
+### Why am I getting a feature flags error on my survey?
+
+```Update survey failed: Cohort with behavioral filters cannot be used in feature flags.
```

Comment:
I'd say it's better to update the error message instead to point to surveys in this case?

Reply:
Agree

Reply:
yes but it still doesn't explain why surveys uses flags